src/js/controller/captions.js

```diff
@@ -1,4 +1,7 @@
-define([], function() {
+define(['../utils/helpers',
+    '../utils/nativerenderingsupported',
+    '../controller/tracksLoader'
```

Comment:
Can we give this files more consistent naming format? Seems we use `kebab-case` most frequently for filenames. Also, could we rename `nativerenderingsupported`? It's a util that does not explicitly identify it to be captions related and it's a little long. Maybe we could do `native-caption` and `tracks-loader` instead?